ERC20_LIST.json

```diff
@@ -372,11 +367,18 @@
         "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/cvx-crvFRAXUSDC.svg",
         "name": "Angle Curve.fi FRAX/USDC Convex Staker",
         "symbol": "cvx-crvFRAXUSDC",
-        "underlyingTokens": [
-          "0x3175Df0976dFA876431C2E9eE6Bc45b65d3473CC"
-        ],
+        "underlyingTokens": ["0x3175Df0976dFA876431C2E9eE6Bc45b65d3473CC"],
         "useInSwap": false,
         "wrappingMethod": "BorrowStaker"
+      },
+      "0x7e880867363A7e321f5d260Cade2B0Bb2F717B02": {
+        "address": "0x7e880867363A7e321f5d260Cade2B0Bb2F717B02",
+        "name": "Staked Curve.fi FRAX/USDC Convex Deposit",
+        "decimals": 18,
+        "symbol": "staked cvxcrvFRAX",
+        "hasPermit": false,
+        "wrappingMethod": "Convex",
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/cvx-crvFRAXUSDC.svg"
```

Comment:
The svg seems off, the logo on the left is missing it seems

Reply:
It does work in the app so I assume it's only github